store/sqlstore/store_test.go

```diff
@@ -617,3 +617,22 @@ func makeSqlSettings(driver string) *model.SqlSettings {
 
 	return nil
 }
+
+func TestExecNoTimeout(t *testing.T) {
+	StoreTest(t, func(t *testing.T, ss store.Store) {
+		sqlStore := ss.(*SqlStore)
+		var query string
+		timeout := sqlStore.master.QueryTimeout
+		sqlStore.master.QueryTimeout = 1
+		defer func() {
+			sqlStore.master.QueryTimeout = timeout
+		}()
```

Comment:
I was a bit worried this could be causing a race condition but didn't spot any and given it was just a test I've committed as is. An alternative would be to setup separate stores just to run this. Let me know if you'd prefer that option.

Reply:
Separate stores would be good, but I'm 0/5. It shouldn't be a big deal.

Reply:
Alright, let's see how it goes. It should be trivial to fix in case of failure.